internal/pkg/storageos/client.go

```diff
@@ -25,40 +26,35 @@ const (
 )
 
 //go:generate mockgen -build_flags=--mod=vendor -destination=mocks/mock_control_plane.go -package=mocks . ControlPlane
-//go:generate mockgen -build_flags=--mod=vendor -destination=mocks/mock_identifier.go -package=mocks . Identifier
```

Comment:
Out of interest why don't we need this mock anymore?

Reply:
That interface (a few line below in that same file) was merged into another interface.
It had a mere 3 method and didn't add anything other than increased complexity - we ended up having another mock obj.